CyberHealth/staticpages/templates/staticpages/accessibility-statement.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% block title %}Accessibility statement - Cyber health assessment{% endblock %}
+{% block content %}
+  <div class="govuk-width-container ">
+    <main class="govuk-main-wrapper govuk-body" id="main-content" role="main">
+      <h1 class="govuk-heading-xl">Accessibility statement</h1>
+
+      <p>
+        This statement applies to content published on the XXXXX domain. It does not apply to content on XXXXX subdomains.
```

Comment:
Could you replace this with:

This statement only applies to the Cyber Health Self Assessment Tool. 

Something like that?